domains/transfer-agreements/TransferAgreements.API/API.IntegrationTests/SwaggerTests.GetSwaggerDoc_AppStarted_NoChangesAccordingToSnapshot.verified.txt

```diff
@@ -344,6 +376,35 @@
         ],
         "type": "string"
       },
+      "ConnectionDto": {
+        "type": "object",
+        "properties": {
+          "id": {
+            "type": "string",
+            "format": "uuid"
+          },
+          "companyId": {
+            "type": "string",
+            "format": "uuid"
+          },
+          "comnpanyTin": {
+            "type": "string"
+          }
```

Comment:
There is no description/summary for any of these properties. That can be done by commenting on the properties of `ConnectionDto`. Maybe you have to change `ConnectionDto` as I do not think it works for a record with positional syntax